applications/polkascan-cli.md

```diff
@@ -0,0 +1,89 @@
+
+#### Language Used
+We will be using **Python** language since we will be needing support of **py-substrate-interface**.
+We will be using **Typer** to implement the CLI: https://github.com/tiangolo/typer.
+
+#### CLI Flags
+
+There are going to be following options when users open up the CLI:
+ - Get current block number.
+ - Get Blocks: Number, time, hash, Extrinsics, Events.
+ - Get block info: Timestamp UTC, Hash, Parent Hash, State root, Extrinsics root, Extrinsics count, Events count, Runtime.
+ - Get Extrinsic details.
+ - Get account info.
+
+### Ecosystem Fit
+ - Where and how does your project fit into the ecosystem?
+Anyone who is looking for account/block/event/transaction details.
+ - Who is your target audience (parachain/dapp/wallet/UI developers, designers, your own user base, some dapp's userbase, yourself)?
+Our target users are ones who wants to take a look at transaction details on blockchain quickly using CLI.
+  
+
+### Simillar platforms
+
+- None so far
+
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+* @avirathecrawler
+
+### Legal Structure
+
+*  **Registered Legal Entity:** Not yet registered
+
+### Team's experience
+
+* avirathecrawler: Did a web3 based startup in early 2016 when ethereum was just launched. Before that, worked with IBM as a software engineer.
+
+### Team Code Repos
+
+* https://github.com/avirathecrawler
+
+## Development Roadmap :nut_and_bolt:
+
+# Overview
+
+## Grant level
+
+Level 1: Up to $10,000, 2 approvals
+
+
+## Development Roadmap :nut_and_bolt:
+
+*  **Total Estimated Duration:** 3 weeks
+
+*  **Total Costs:** 5,000 USD
+
+### Milestone 1
+
+*  **Estimated Duration:** 2 weeks
+
+*  **Costs:** 5,000 USD
+
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 0a. | Apache License 2.0 | All code will be published under Apache 2.0 |
+| 0b. | Documentation | We will provide inline documentation of the code. |
+| 0c. | Testing and it's Guide | Core functions will be fully covered by comprehensive unit tests written in PyTest and CLI tests in PyTest to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Articles | We will publish an article on how to contribute and use this tool. |
+| 0. | Creation CLI tool | CLI tool will be able to fetch account, block, event, transaction and logs as on explorer.We will be running our own instance of polkascan.io https://github.com/polkascan/explorer. CLI will be coded in Python since we might be using py-substrate-interface: https://github.com/polkascan/py-substrate-interface and polkascan API which will be provided from our own instance. CLI will be having interactive menu where user can navigate using arrow keys. |
```

Comment:
Sorry, I actually have two more comments:

1. You may want to change the name from Polkascan-CLI, as this might imply that the Polkascan Foundation is behind this. 
2. Why is there is one milestone of 2 weeks but the total time is given as 3 weeks?
3. Why have an interactive menu? Based on earlier comments, I thought this was entirely command-line-driven. I don't think an interactive menu is necessary; someone who wants interactivity could use a browser.

Reply:
Hey @laboon.
I have changed the name.
Fixed the timeline.
By interactive, I meant the user would be able to navigate the CLI using arrow keys. Anyways, fixed it.